root/src/main/java/com/example/servingwebcontent/ConnectDatabase.java

```diff
@@ -0,0 +1,58 @@
+package com.example.servingwebcontent;
+
+import java.sql.*;
+import java.util.*;
+
+public class ConnectDatabase {
+	Connection con = null;
+
+	public ConnectDatabase () {
+	}
+
+	public ConnectDatabase (String database, String user, String pass) {
+		try {
+			con = DriverManager.getConnection("jdbc:mysql://localhost/" + database + "?" + "user=" + user + "&password=" + pass);
+		} catch (SQLException e) { System.out.println(e); }
```

Comment:
you should handle the exception, not just print it out. Same for all the other exceptions you catch in this file.

Reply:
Throwing a RuntimeException with an appropriate error message is probably good enough in our case.